mantidproject/mantid
# Add adjustments to reconstructed data in Maxent

**Description of work.**

**To test:**

Run the algorthm with the new properties DataLinearAdj, DataConstAdj & PerSpectrumReconstruction and check it behaves as expected. Also build and check the documentation of Maxent.

I suggest running variations of the two usage examples added to the documentation to the documentation to do the testing.

Fixes #21577 

Does this update require release notes?
 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
